### PR TITLE
Matt/route detail

### PR DESCRIPTION
**Pull request by mattbarker016**
Somehow included everyone's commits and files :/

### CODE REVIEW

**Comment by mattbarker016** [comment]
I merged the conflicts myself. Changes were mainly made in RouteDetailViewController. I also added a dummy JSON file, and added the new Network base url

File: TCAT/Controllers/RouteDetailViewController.swift [hunk 1]
```diff
@@ -124,22 +130,25 @@ class RouteDetailViewController: UIViewController, GMSMapViewDelegate, CLLocatio
...
-        }
+        }*/
         
         // Set up Location Manager
         locationManager.delegate = self
         locationManager.desiredAccuracy = kCLLocationAccuracyNearestTenMeters
         locationManager.distanceFilter = 10
         locationManager.requestWhenInUseAuthorization()
         locationManager.startUpdatingLocation()
+
```
Comment (AAAstorga) [inline TCAT/Controllers/RouteDetailViewController.swift:151]: Get rid of extra line

File: TCAT/Controllers/RouteDetailViewController.swift [hunk 2]
```diff
@@ -207,46 +213,21 @@ class RouteDetailViewController: UIViewController, GMSMapViewDelegate, CLLocatio
...
-        // Key Assumption: map is oriented where north is "up"
-        
-        // average between min and max
-        let newLong = CLLocationDegrees((minLong + maxLong) / 2) // average
-        // double longest veritcal distance, plus multiplication by constant to simulate padding
-        let newLat = CLLocationDegrees(0.99995 * (maxLat - 2 * abs(maxLat - minLat)))
-        
-        bounds = bounds.includingCoordinate(CLLocationCoordinate2D(latitude: newLat, longitude: newLong))
-        
+
```
Comment (AAAstorga) [inline TCAT/Controllers/RouteDetailViewController.swift:229]: Extra space can be deleted

**Review by AAAstorga — APPROVED** [review]
Everything else looks good! After changes get this baby into master

**Pull request closed without merging**